pbench-relay-server.service

```diff
@@ -0,0 +1,34 @@
+# file-relay service for Pbench
+# Stolen from the pbench-server service file which was originally generated by Podman
+
+[Unit]
+Description=Containerized file-relay for Pbench
+Documentation=man:podman-generate-systemd(1)
+Wants=network-online.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStart=/usr/bin/podman run \
+	--cidfile=%t/%n.ctr-id \
+	--cgroups=no-conmon \
+	--rm \
+	--sdnotify=conmon \
+	--replace \
+	--detach \
+	--name pbench-file-relay \
+	--network host \
+	--volume /srv/RELAY:/srv/RELAY:Z \
+	quay.io/pbench/file-relay:v0.1.0 \
+	--server_id RELAY --bind 10.1.170.231:8002 --files-directory /srv/RELAY
```

Comment:
We really shouldn't be submitting internal host IP addresses in GitHub! There's a `%H` systemd substitution for "hostname", although I'm not entirely sure of the translation: it's not very specific. (And if it's just `$(hostname)` then it's probably no use as that's just the VM name not a routable address.) Still, having the literal IP here isn't a good idea. If we can't get something that'll actually work, a placeholder like `<host_name>` would be best. Or maybe `0.0.0.0` would work here?

Reply:
I thought you might object to this.  😆

> We really shouldn't be submitting internal host IP addresses in GitHub!

I agree, but the `10.0.0.0/8` subnet is not unique to RH -- it's a "private" subnet which is available for use by anyone, and it is "non-routeable" (over the open internet, at least) -- so it's not actually an "internal host IP address" and no one outside the RH network can reach _our_ node which is using it (they'll get sent to some other node local to them).

I basically just saved the configuration that we're using internally, so that I don't have to recreate it.  For our external configuration, I expect to omit the address from the `--bind` option (it defaults to `0.0.0.0`) and specify just the port, and I'll omit the `--files-directory` option and let it default to using container storage.  (I selected the options above because I wanted to create a drop-in replacement for what you had set up.)

That said, if you feel strongly, I can remove the IP address and leave just the colon and port...let me know.

Reply:
While it's true that the internal NAT address isn't directly reachable from outside, it's advertising a specific target inside, and it just seems like a bad idea when there's no real reason for it. So, yeah, I'd prefer that it be changed, either to a simple `0.0.0.0` or to a placeholder.